packages/expo-video/android/src/main/java/expo/modules/video/player/VideoPlayer.kt

```diff
@@ -378,7 +378,10 @@ class VideoPlayer(val context: Context, appContext: AppContext, source: VideoSou
 
   private fun sendEvent(event: PlayerEvent) {
     // Emits to the native listeners
-    event.emit(this, listeners.mapNotNull { it.get() })
+    val listenersSnapshot = listeners.mapNotNull { it.get() }
+
+    event.emit(this, listenersSnapshot)
+
```

Comment:
Correct me if I'm wrong, but I'm pretty sure that this should make no functional difference between the original implementation. We can still perform a modification while iterating with `mapNotNull`. We should do this like above instead:
```suggestion
    val listenersSnapshot = listeners.toList().mapNotNull { it.get() }

    event.emit(this, listenersSnapshot)
```